web/src/components/tasks/business-formation/getErrorStateForFormationField.ts

```diff
@@ -51,7 +51,10 @@ export const onlyHasErrorIfUndefined: FormationFields[] = [
   "businessSuffix",
 ];
 
-export const getErrorStateForField = (inputParams: {
+export const BUSINESS_ADDRESS_LINE_1_MAX_CHAR = 35;
+export const BUSINESS_ADDRESS_LINE_2_MAX_CHAR = 35;
```

Comment:
This is another thing that could get moved into a consolidated address helpers file in the future.

Reply:
moved variables to that file